src/components/templates/ExternalExperience/ExternalExperience.module.scss

```diff
@@ -39,25 +46,37 @@
     justify-content: flex-start;
     align-items: flex-start;
     padding: 0;
+    padding-bottom: space.empty(0.83);
```

Comment:
it's for 10px. I've seen similar in the code, so decided that it'd be ok (?)

Reply:
I don't think it's ok to ask for `10px` since there is `space.empty()` to be unifying, and there are
```scss
$s2: 2px;
$xs: 4px;
$sm: 8px;
$md: 12px; // previously named $whitespace
$lg: 16px;
$xl: 24px;
$l2: 32px;
$l3: 42px;
```
in `space.scss`.

These are meant to reduce every single component having their own margins, paddings, widths and heights that make the overall UI incoherent.

Reply:
AND, of course, there are `space.gap()` and `space.button()`